### DISCUSSION TITLE
Android 11: Application Installation Failed

### DISCUSSION THREAD

**Discussion by vsevjednom-cz** [OpenTracksApp/OpenTracks#885] [Q&A] [answered]
I generate apk file in `Android studio` - as usually. Everything works without problem in Android 9 and older. But installation fails in Android 11. The error is `Aplikaci nelze nainstalovat` in czech language. It's `Application Installation Failed` in english - I think. I haven't got more details because I know about this error from my friend.
I searched on the internet. Android 11 needs APK Signature Scheme v2 and I can check actual signature with `apksigner verify`. The result of the check is weird:
```
DOES NOT VERIFY
ERROR: Missing META-INF/MANIFEST.MF
```
So I tried `Generate signed APK` in `Android studio`. But i don't see any file. I'm still searching and reading in manual but without success.  Could you help me what I must to do for sign apk file?

**Comment by dennisguse** [comment, marked as answer]
Build -> Generated signed bundle/apk and then select v1+v2.
That's how I build OpenTracks for the PlayStore.
Was never a problem.

![image](https://user-images.githubusercontent.com/2820825/125840753-3d381ced-bfd7-4fa6-8e37-ad3fcc0dad45.png)

**Comment by dennisguse** [comment]
The APK is a folder named by the build variant: 
![image](https://user-images.githubusercontent.com/2820825/125840911-ab8ff6cf-bdc6-47f3-a004-fd0aaf983287.png)

**Comment by dennisguse** [comment]
PS: You might get better error messages via Logcat, but not via the UI.
It basically means: "doens't work" :)
